doc/source/parameters/cfd/mesh_adaptation_control.rst

```diff
@@ -0,0 +1,60 @@
+Mesh Adaptation Control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This subsection controls the mesh adaptation method, with default values given below.
+
+.. code-block:: text
+
+	subsection mesh adaptation
+	  # Type of mesh adaptation. Choices are  none, uniform or kelly.
+	  set type                 = none
+
+	  # Variable for kelly estimation. Choices are velocity, pressure or phase.
+	  set variable             = velocity
+
+	  # Frequency of the mesh refinement
+	  set frequency            = 1
+
+	  # Minimum refinement level
+	  set min refinement level = 0
+
+	  # Maximum refinement level
+	  set max refinement level = 10
+
+	  # Fraction of coarsened elements
+	  set fraction coarsening  = 0.05
+
+	  # Fraction of refined elements
+	  set fraction refinement  = 0.1
+
+	  # How the fraction of refinement/coarsening are interpreted
+	  # Choices are number or fraction 
+	  set fraction type        = number
+
+	  # Maximum number of elements
+	  set max number elements  = 100000000
+	end
+
+* Two ``type`` of mesh adaptation are available. The ``uniform`` mesh adaptation refines the mesh at every cell, whereas the ``kelly`` uses a `kelly error estimator <https://www.dealii.org/current/doxygen/deal.II/classKellyErrorEstimator.html>`_ to decides which cell are refined, by estimating the error per cell for a given variable. The variable for kelly estimation should be specified with ``set variable``, and can be:
+	* velocity
+	* pressure
+	* phase (for multiphase flows)
+* The frequency at which the mesh is refined is controlled with the ``frequency`` parameter. If ``set frequency = 1``, the mesh is refined at every time step.
```

Comment:
I am absolutely not sure, but I think it has to do with the number of mesh refinement more than time steps. Let's say for example if the number of mesh refinement is 4, I think this frequency sets at which of these mesh refinement there will be mesh adaptation but this is a total guess. @blaisb ?

Reply:
is it a difference between steady and transient simulation? I'll wait for Bruno's input on that :)

Reply:
It's the frequency at which the mesh is refined. The number of refinement in a steady-state simualtion is controlled by the number mesh adapt parameter in the simulation control :).